### ISSUE TITLE
When changing data label colors, categories won't show

### ISSUE THREAD

**Issue by emmartin-design** [scanny/python-pptx#650]
Hello!

If I set category names to show, without changing anything else, categories show as expected.
`chart.plots[0].data_labels.show_category_name = True`

However, because I need some data labels to be white, and some to be dark due to the background they appear over, I need to set up code that goes point by point and changes the color.  When this is done, the categories will not show. The following is the code I'm currently using to achieve this.

`def differing_data_labels(chart, ic, nos, noc):  # Uses colorbreak to assign label font color point by point
    for series_color, series in enumerate(chart.series):
        for point_color, point in enumerate(series.points):
            pointcolor = point.data_label.font.color
            pointcolor.theme_color = MSO_THEME_COLOR.TEXT_1
            if ic == 'PIE':
                point.data_label.position = XL_LABEL_POSITION.BEST_FIT
                cbreak = colorbreak(noc)
                if point_color < cbreak:
                    pointcolor.theme_color = MSO_THEME_COLOR.BACKGROUND_1
                point.data_label.show_category_name = True
            else:
                cbreak = colorbreak(nos)
                if series_color < cbreak:
                    pointcolor.theme_color = MSO_THEME_COLOR.BACKGROUND_1`

No matter where I place the command to show the categories, if the points have been iterated over, it seems to override the command and no categories will show. Other such commands, such as show_data_labels, do not have this problem.

**Comment by scanny** [comment]
Yeah, on some of these finer points PowerPoint can be a bit fussy. At this low level I would do what I want using PowerPoint and then inspect the XML it emits to accomplish that, then compare it with what I'm getting with the `python-pptx` API and see how they differ.

Then you can address either figuring out how to produce that result with the calls `python-pptx` gives you or getting as close as you can with `python-pptx` and setting the last few attributes using `lxml` methods.
